server/consumer.go

```diff
@@ -1591,8 +1623,62 @@ func (o *consumer) updateConfig(cfg *ConsumerConfig) error {
 		}
 	}
 
-	// Record new config for others that do not need special handling.
-	// Allowed but considered no-op, [Description, SampleFrequency, MaxWaiting, HeadersOnly]
+	singleFilterChanged := false
```

Comment:
var singleFilterChanged bool

Reply:
simplified, so no need for this declaration at all now.